pixeltable/catalog/table_version.py

```diff
@@ -1431,6 +1431,21 @@ def unlink_external_store(self, store: ExternalStore) -> None:
         self._tbl_md.external_stores.pop(idx)
         self._write_md(new_version=True, new_schema_version=True)
 
+    @property
+    def cloud_uri(self) -> str | None:
+        return self._tbl_md.additional_md.get('cloud_uri')
```

Comment:
Do we have a defined schema for additional_md somewhere? (pxt_uri would be a better name, just like we have s3_uri, etc.)

Reply:
we can rename it to pxt_uri 
For pixeltable core, we dont have schema defined we add TypedDict that has one entry pxt_uri

for cloud we store it as dict but its managed from cloud side directly (that schema is not visible here)

Reply:
We could add a defined schema, but maybe as a follow up?